packages/static-wado-webserver/lib/controllers/serverSCPController.mjs

```diff
@@ -0,0 +1,35 @@
+import dicomWebServerConfig from "../dicomWebServerConfig.mjs";
+import { findMove } from "../subscribers/dimse/operations/cmove.mjs";
+
+export default function serverSCPFallbackController(
+  { studyInstanceUIDPattern, seriesInstanceUIDPattern, sopInstanceUIDPattern } = {
+    studyInstanceUIDPattern: "",
+    seriesInstanceUIDPattern: "",
+    sopInstanceUIDPattern: "",
+  },
+  bulk = false
+) {
+  return async (req, res, next) => {
+    try {
+      const requestOptions = {
+        bulk,
+        destAETittle: dicomWebServerConfig.dicomWebScpConfig.scpAe,
+        StudyInstanceUID: req.params[studyInstanceUIDPattern],
+        SeriesInstanceUID: req.params[seriesInstanceUIDPattern],
+        SOPInstanceUID: req.params[sopInstanceUIDPattern],
+      };
+
+      if (bulk) {
+        const [findResults] = await findMove(dicomWebServerConfig.sourceScpConfig, requestOptions);
```

Comment:
This should come from the params, not a hard coded reference to a specific config.  Otherwise, you break the ability to have proxies for two different back ends.  Not saying we are using it now, but it is definitely a requirement.